variables.tf

```diff
@@ -627,25 +642,25 @@ variable "client_region" {
 
 variable "client_instance_metadata" {
   description = "Metadata key/value pairs to make available from within the client instance."
-  type        = "map"
+  type        = map(string)
   default     = {}
 }
 
 variable "client_ssh_allow_ranges" {
   description = "List of CIDRs that will be allowed ssh access to forseti client"
-  type        = "list"
+  type        = list(string)
   default     = ["0.0.0.0/0"]
 }
 
 variable "client_tags" {
   description = "GCE Forseti Client VM Tags"
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
 variable "client_access_config" {
   description = "Client instance 'access_config' block"
-  type        = "map"
+  type        = map(string)
```

Comment:
This should be an object since there is a static set of expected keys.

Reply:
Fixed